src/signing/networkMessages/kusama.js

```diff
@@ -43,7 +43,9 @@ export async function StakeTx(
   if (to.length > 0) {
     // only controller addresses can nominate (for not set controllers, we set the controller above)
     if (["controller", "stash/controller", "none"].includes(addressRole)) {
-      const response = await api.query.staking.nominators(senderAddress)
+      const stakingLedger = await api.query.staking.ledger(senderAddress)
```

Comment:
nice! but better to get this from the API?

Reply:
currently the controller will not show any validators in the first place

Reply:
you merged #4016 with the exact same logic xD

Reply:
I retract. Let's do this for now and later get the validator list from the API